content/en/news/releases/1.19.x/announcing-1.19/upgrade-notes/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Istio 1.19 Upgrade Notes
+description: Important changes to consider when upgrading to Istio 1.19.0.
+weight: 20
+publishdate: 2023-09-01
+---
+
+When you upgrade from Istio 1.18.x to Istio 1.19.0, you need to consider the changes on this page.
```

Comment:
Again, this would really apply to any 1.19 release. ~~I think we can just go with 1.18 and 1.19 meaning all releases similar to the announce page and above.~~ I would also be fine changing this to 1.19.x to keep the two values constant. We seem to use .x a bunch below so changing to use that, but removing the .x is OK with me. Not sure how others feel.
```suggestion
When you upgrade from Istio 1.18.x to Istio 1.19.x, you need to consider the changes on this page.
```

Reply:
I agree with either `.x` or leaving it off versus a .0, since the upgrade will apply to all patch versions. I don't have a strong preference of 1.19 versus 1.19.x, though.